op-e2e/actions/op_program_test.go

```diff
@@ -19,12 +26,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var dumpFixtures = false
+var (
+	dumpFixtures = false
+	fixtureDir   string
+)
 
 func init() {
 	if os.Getenv("OP_E2E_DUMP_FIXTURES") == "1" {
 		dumpFixtures = true
 	}
+	fixtureDir = os.Getenv("OP_E2E_FPP_FIXTURE_DIR")
```

Comment:
Maybe we just dump fixtures if `OP_E2E_FPP_FIXTURE_DIR` is set so we don't need two separate env vars?